src/util/ChartUtils.ts

```diff
@@ -847,14 +863,16 @@ export const getStackedData = (data: any, stackItems: any, offsetType: string) =
   return stack(data);
 };
 
+export type StackGroups = any;
```

Comment:
I do not find that this adds value. What is the intention?

Reply:
Documentation - sort of "I don't know what this type is exactly but it's the same as over there". Bringing a bit of information into the sea of `any`.

Reply:
Fair enough. Could you please add a small comment as to what a StackGroup is?